src/sentry/integrations/vercel/integration.py

```diff
@@ -170,13 +192,30 @@ def build_integration(self, state):
                 "installation_id": data["installation_id"],
                 "installation_type": installation_type,
                 "webhook_id": webhook["id"],
+                "configurations": configurations,
+            },
+            "post_install_data": {
+                "user_id": state["user_id"],
+                # new configuration data
+                "configuration_id": data["installation_id"],
+                "access_token": access_token,
+                "webhook_id": webhook["id"],
             },
-            "post_install_data": {"user_id": state["user_id"]},
         }
 
         return integration
 
     def post_install(self, integration, organization, extra=None):
+        # add new configuration information to metadata
+        configurations = integration.metadata.get("configurations") or {}
+        configurations[extra["configuration_id"]] = {
+            "access_token": extra["access_token"],
```

Comment:
@MeredithAnya This is the same access token as in `integration.metadata`, yea? Why do we need to add it to `extra`?

Reply:
i know that its duplicating the information but my thought was to have it already in the configurations so that the next time its already formatted correctly and we can have the org already.

Reply:
@MeredithAnya My issue is not with having it in the configurations, it's with having to put this in `post_install_data`. You can populate `configurations` from `integration.metadata`.

Reply:
In fact, I think you can take care of all of the logic related to this change in `build_integration`.

Reply:
@scefali hmm yeah okays i see what you mean. yeah lemme update that. we still need the post install to update adding the organization id though

Reply:
@MeredithAnya Oh yea, forgot about the `organization_id`. So it probably makes sense to keep the update here but pull from metadata instead of extra.